apis/datadoghq/v2alpha1/datadogagent_default.go

```diff
@@ -209,6 +210,10 @@ func defaultFeaturesConfig(ddaSpec *DatadogAgentSpec) {
 
 	apiutils.DefaultBooleanIfUnset(&ddaSpec.Features.Dogstatsd.OriginDetectionEnabled, defaultDogstatsdOriginDetectionEnabled)
 
+	if *ddaSpec.Features.Dogstatsd.OriginDetectionEnabled {
+		apiutils.DefaultStringIfUnset(&ddaSpec.Features.Dogstatsd.TagCardinality, defaultDogstatsdTagCardinality)
+	}
+
```

Comment:
do you mind removing this so that the default setting will fallback to the agent?